.github/workflows/test-action.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         path: ${{ env.ARTIFACTS_DIR }}
 
     - id: metrics-reporter
-      uses: tier4/ros-metrics-reporter@main
+      uses: tier4/ros-metrics-reporter@feature/change-summary
```

Comment:
Can you use `uses: ./` like this? :thinking: 
https://github.com/ros-tooling/action-ros-ci/blob/a1c6a4d5473444b5f3a6bfa66fedc233a1e6e5a1/.github/workflows/test.yml#L71

Reply:
Thanks! It works!
The reason I thought that private action couldn't be used with Composite action was because there was no mention of private action only in the Composite action documentation [here](https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action#testing-out-your-action-in-a-workflow).
Today I was able to notice this. I'm going to submit a fix request to the GitHub documentation.

Reply:
hmm? What is `private action`? This repository is public. :thinking:

Reply:
Please refer to this document. This is the JavaScript action documentation.
The method for testing Composite action is also divided into private action and public action.
https://docs.github.com/ja/actions/creating-actions/creating-a-docker-container-action#testing-out-your-action-in-a-workflow

Reply:
Okay, thank you!

Reply:
The link you wanted to point out is... this? :thinking: 
https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action

Reply:
Yes, the English version of the documentation is a link to your comment.